data/transactions/logic/debugger_test.go

```diff
@@ -125,3 +125,28 @@ func TestDebuggerHook(t *testing.T) {
 	require.Greater(t, testDbg.update, 1)
 	require.Equal(t, 1, len(testDbg.state.Stack))
 }
+
+func TestLineToPC(t *testing.T) {
```

Comment:
Add missing test case for LineToPC(`negative value`) and a high value integer, i.e. 0x7fffffff

Reply:
Added.